src/voiceController/VoiceController.js

```diff
@@ -1,95 +1,111 @@
-import React, { createContext, useEffect, useState } from "react";
+import React, { createContext, useEffect, useState } from 'react'
+import { getInteractionType, getMatchedWord, getValue } from './utils'
+import { ineractionTypes } from './constants'
```

Comment:
man just a typo, ineraction -> interaction. Have to update everwhere it is used though